Manual/manJHUGenerator.tex

```diff
@@ -301,6 +301,8 @@ \section{Installation}
 
 By default, the makefile options \verb|linkMELA| and \verb|linkCollierLib| are set to \verb|No|. If you need to use the NLO features, you need to set \verb|linkCollierLib = Yes|, and download and compile the COLLIER package using \verb|./downloadcollier.sh| with optional arguments \verb|compiler| and \verb|-j| to specify the Fortran compiler and the number of cores to use for compilation, respectively. Likewise, if you need to use off-shell processes, which use modified MCFM matrix elements, you need to set \verb|linkMELA = Yes| and compile JHUGenMELA with at least the option \verb|deps| before compiling JHUGenerator (see compilation instructions for JHUGenMELA for more details). In the case \verb|linkMELA = Yes|, the compiler has to be set to \verb|Comp = gfort|, and the gcc version should be at least 4.6. Notice that JHUGenMELA has its own compilation routine and location for the COLLIER library.
 
+Also note that after JHUGenerator v7.3.0 was released, the naming scheme used for COLLIER libraries was changed. For any version of JHUGenerator preceding and including v7.3.0, this change in package location must be accounted for before installing the COLLIER library. If you are compiling JHUGenMELA, the script \verb|COLLIER/setup.sh| must be edited to use \verb|pkgname = "collier-1.2.0"|. In the case you do not link JHUGenMELA but still need COLLIER, \verb|./downloadcollier.sh| should reference \verb|collier-1.2.0.tar.gz| instead of \verb|collier-1.2.tar.gz|.
```

Comment:
Please add an explicit statement to mention that this change is not needed for the later versions.

Reply:
we can simply add a sentence at the end of the paragraph: “None of these manual edits are needed for the JHUGenerator version 7.4.0 and above.” 
but this could be done later, in a later iteration, no need to delay this merging. It is already written pretty clearly that it is for versions below.

Reply:
This could be done instead of making this comment... Such an addition would be fine with me.

Reply:
Lets move forward, it is not code to be reviewed

Reply:
PRs are not just for code; we had other PRs pending that did not feature code changes. This is a normal part of a transparent review.

Reply:
You had many days to make your comment, why block it now? This is absolutely not important.

Reply:
No, this was actually a question I raised. As you could read from my comment, I did not make any specific suggestion to change anything. My comment was only a question on what was actually done (but ofc if we find out what was done is different, we should make the appropriate change, which could be later indeed). The other comment is a specific change request, so that one should be done.

Reply:
There is no "must." The manual is very clear with the words: "For any version of JHUGenerator preceding and including v7.3.0..." So, no change is needed to make it clearer. Lets move forward.